pkg/node/node.go

```diff
@@ -1233,21 +1233,29 @@ func (s *Service) NodeGetInfo(ctx context.Context, _ *csi.NodeGetInfoRequest) (*
 					log.Errorf("couldn't get targets from array: %s", err.Error())
 					continue
 				}
-
+				var ipAddress string
 				var iscsiTargets []goiscsi.ISCSITarget
 				for _, address := range infoList {
 					// first check if this portal is reachable from this machine or not
 					if ReachableEndPoint(address.Portal) {
+						ipAddressList := strings.Split(address.Portal, ":")
```

Comment:
We may not yet support IPv6 but what's the impact here? is using colons the right choice of separator?